### ISSUE TITLE
Support for mbart models?/ Could we get the output logits of decoders before the beam search?

### ISSUE THREAD

**Issue by leoozy** [NVIDIA/FasterTransformer#121]
The mbart model is implemented by hugging face

**Comment by byshiue** [comment]
It requires to modify the source codes.

**Comment by leoozy** [comment]
@byshiue Thank you for your reply. If I just use the python interface, could I get the logits output of the decoder?

**Comment by byshiue** [comment]
If you use decoder op, then the output is the results of transformer block.
If you use decoding op, then you need to modify the op and FT source codes.

**Comment by leoozy** [comment]
@byshiue Thank you. I will try to implement the mbart model
